src/main/java/gregtech/api/capability/impl/miner/MinerLogic.java

```diff
@@ -282,7 +282,8 @@ private void mineAndInsertItems(NonNullList<ItemStack> blockDrops, WorldServer w
         // If the block's drops can fit in the inventory, move the previously mined position to the block
         // replace the ore block with cobblestone instead of breaking it to prevent mob spawning
         // remove the ore block's position from the mining queue
-        if (GTTransferUtils.addItemsToItemHandler(metaTileEntity.getExportItems(), true, blockDrops)) {
+        if (GTTransferUtils.addItemsToItemHandler(metaTileEntity.getExportItems(), true, blockDrops) ||
+                metaTileEntity.getExportItems().getSlotLimit(0) == Integer.MAX_VALUE - 1) { // Check for ME Output
```

Comment:
Can this result in voiding?

Also why does the ME output bus work with `AbstractRecipeLogic` but not here? They both do the same check with the same method.